app/src/main/java/org/p2p/wallet/newsend/model/RecipientConverter.kt

```diff
@@ -0,0 +1,80 @@
+package org.p2p.wallet.newsend.model
+
+import org.p2p.core.token.Token
+import org.p2p.core.token.TokenData
+import org.p2p.core.token.TokenVisibility
+import org.p2p.core.utils.toBigDecimalOrZero
+import org.p2p.wallet.home.db.TokenEntity
+import org.p2p.wallet.newsend.db.RecipientEntity
+import org.p2p.wallet.send.model.AddressState
+import org.p2p.wallet.send.model.SearchResult
+import java.util.Date
+
+private const val EMPTY_TIMESTAMP = 0L
+
+object RecipientConverter {
```

Comment:
class
naming is RecipientRepositoryMapper

Reply:
<img width="707" alt="Screen Shot 2022-12-21 at 21 01 36" src="https://user-images.githubusercontent.com/100791035/208911488-a4b63578-8632-402e-acd1-f6ec3b3e0b68.png">

Reply:
![image](https://user-images.githubusercontent.com/99185280/208911766-163e0173-94ac-45c0-958d-3edfb870cad8.png)
no i see TokenConverter

Reply:
old naming, but okay